### PR TITLE
CLDC-2164 Bugfix on rent ranges validate when >= 4 beds

### PR DESCRIPTION
**Pull request by ghost**
https://digital.dclg.gov.uk/jira/browse/CLDC-2164

### CODE REVIEW

File: app/models/validations/financial_validations.rb [hunk 1]
```diff
@@ -187,7 +187,12 @@ def validate_rent_range(record)
 
     collection_year = record.collection_start_year
 
-    rent_range = LaRentRange.find_by(start_year: collection_year, la: record.la, beds: record.beds_for_la_rent_range, lettype: record.lettype)
+    rent_range = LaRentRange.find_by(
+      start_year: collection_year,
+      la: record.la,
+      beds: record.beds_for_la_rent_range,
+      lettype: record.lettype,
+    )
```
Comment (arfacamble) [inline app/models/validations/financial_validations.rb:195]: 🎉

File: app/models/validations/financial_validations.rb [hunk 2]
```diff
@@ -200,7 +205,9 @@ def validate_rent_range(record)
         record.errors.add :rent_type, I18n.t("validations.financial.brent.rent_type.below_hard_min")
         record.errors.add :needstype, I18n.t("validations.financial.brent.needstype.below_hard_min")
         record.errors.add :period, I18n.t("validations.financial.brent.period.below_hard_min")
-      elsif record.beds.blank? || record.beds < LaRentRange::MAX_BEDS
+      end
+
+      if record.weekly_value(record["brent"]) > rent_range.hard_max
```
Comment (arfacamble) [inline app/models/validations/financial_validations.rb:210]: this looks sensible

**Review by arfacamble — APPROVED** [review]

**Review by natdeanlewissoftwire — APPROVED** [review]

File: spec/models/validations/financial_validations_spec.rb [hunk 3]
```diff
@@ -788,7 +788,7 @@
           LaRentRange.create!(
             ranges_rent_id: "1",
             la: "E07000223",
-            beds: 1,
+            beds: 4,
```
Comment (jamesrose) [inline spec/models/validations/financial_validations_spec.rb:791]: Please could you add a test for > 4 beds?

File: app/models/validations/financial_validations.rb (hunk 2 above)
Comment (jamesrose) [reply app/models/validations/financial_validations.rb:210]: Is it possible for there to be a `below_hard_min` and `above_hard_max` at the same time? That seems unlikely.

**Pull request merged**